include/LightGBM/config.h

```diff
@@ -158,6 +158,7 @@ struct Config {
   // desc = ``rf``, Random Forest, aliases: ``random_forest``
   // desc = ``dart``, `Dropouts meet Multiple Additive Regression Trees <https://arxiv.org/abs/1505.01866>`__
   // descl2 = **Note**: internally, LightGBM uses ``gbdt`` mode for the first ``1 / learning_rate`` iterations
+  // desc = ``mvs``, Minimal variance sampling <https://arxiv.org/abs/1910.13204>__
```

Comment:
```suggestion
  // desc = ``mvs``, `Minimal variance sampling <https://arxiv.org/abs/1910.13204>`__
```

I think these extra backticks are necessary for this to format correctly as a hyperlink.

See

https://github.com/microsoft/LightGBM/blob/61e464bc02bd325672ce71daabfa57d569cf02bc/include/LightGBM/config.h#L159

and

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages